### PR TITLE
Bug 1178696 - Making the rocketbar input width consistent across apps r=kgrandon

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=47cadcdcddbf93d1753f64de81f4810a1264199e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=627386182a98a6f242a844dd467f982f6d127e4a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8dfd8dfa5fafc4e26984a4614a15a01568eb13a6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0698291a773a0ff4e8afd43b1b56dd5b6dc48b4c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d63a293865b5a678a0678ceb2e6686dfa1801396

File: apps/system/style/app_titlebar.css [hunk 1]
```diff
@@ -61,6 +61,10 @@
   position: absolute;
 }
 
+.appWindow:not(.collapsible) .chrome + .titlebar .titlebar-minimized {
```
Comment (KevinGrandon) [inline apps/system/style/app_titlebar.css]: Since we're touching this, can you add a comment about what the 11.5 rem is for?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=da6723023a525efe7d774b3f6d9eb8ac04c233f5

**Pull request merged**